.renovaterc.json

```diff
@@ -76,6 +76,14 @@
       "paths": ["+(package.json)"],
       "updateTypes": ["minor"],
       "groupName": "core dependencies (minor updates)"
+    },
+
+    {
+      "assignAutomerge": true,
+      "automerge": true,
+      "updateTypes": ["minor", "patch"],
+      "packagePatterns": ["^@ampprojects/"],
+      "groupName": "ampproject dependencies (minor & patch updates)"
```

Comment:
We currently disable updates for everything under `dependencies`. Does that rule need to be modified for this group to take effect?

https://github.com/ampproject/amphtml/blob/3c4af2940d1a40e8541b2d643c4db6a22af6daaa/.renovaterc.json#L8-L10

Reply:
You are indeed correct; fixed. Some of the subgroups may merit enabling dependencies, albeit in a follow-up PR/discussion.

Reply:
Absolutely! I think the runtime WG will be happy to see PRs directed to them for most of these dependencies:

https://github.com/ampproject/amphtml/blob/e1a66bce763efff45522ab77e15cdc7ff2aaebc7/package.json#L21-L29

Context: https://github.com/ampproject/amphtml/issues/24784#issuecomment-536339301